docs/FAQ/how-to-manually-configure-ssl-using-cert-manger-in-bold-bi-kubernetes-deployment.md

```diff
@@ -0,0 +1,109 @@
+# How TO Manually Configure SSL Using Cert Manager in Bold BI Kubernetes Deployment
+
+This section describe how to manually configure ssl using cert manager on kubernetes. Currently we have provided support for Nginx and Istio as Load Balancers in Bold BI. Please refer the below links for configure ssl based on your load balancer.
+
+* [Configure SSL Using Cert Manager in Bold BI Kubernetes Deployment with Nginx](#configure-ssl-using-cert-manager-in-bold-bi-kubernetes-deployment-with-nginx)
+
+* [Configure SSL Using Cert Manager in Bold BI Kubernetes Deployment with Isito](#configure-ssl-using-cert-manager-in-bold-bi-kubernetes-deployment-with-istio)
+
+## Configure SSL Using Cert Manager in Bold BI Kubernetes Deployment with Nginx
+
+Follow the below steps to configure ssl when using Nginx load balancer in Bold BI kubernetes deployment.
+
+1. To deploy the Cert-manager in your cluster create a namespace and deploy it using helm or kubectl.
+	
+	Command for deploy Cert-manager using helm:
+	
+	```console
+	helm install cert-manager jetstack/cert-manager --namespace cert-manager --create-namespace --version v1.10.0 --set installCRDs=true --set global.leaderElection.namespace=cert-manager
+	
+	```
+
+	Command for deploy Cert-manager using kubectl:
```

Comment:
Command for deploying Cert-Manager using kubectl:

Reply:
concern addressed